as-ap-currinfo/scripts/sirius-ioc-ts-ap-currinfo.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/local/bin/python-sirius -u
+"""TS-AP-CurrentInfo IOC executable."""
+
+from as_ap_currinfo import as_ap_currinfo as ioc_module
+ioc_module.run('TS')
```

Comment:
can we use lowercase here? just for standardization

Reply:
done!